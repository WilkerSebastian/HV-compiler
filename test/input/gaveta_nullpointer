0-50 130 031 000