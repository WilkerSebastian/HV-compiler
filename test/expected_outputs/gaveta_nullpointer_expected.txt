Erro na leitura do gaveteiro no endereÃ§o 31, tentativa de leitura em endereÃ§o inexistente
